pepper-apis/src/main/java/org/broadinstitute/ddp/util/TemplateRenderUtil.java

```diff
@@ -18,4 +20,18 @@ public static String toPlainText(Long templateId, Renderable.Provider<String> re
         }
         return renderedValue;
     }
+
+    public static String applyRenderedTemplate(Long templateId, String templateText,
+                                               Renderable.Provider<String> rendered, ContentStyle style) {
+        if (templateId != null) {
+            templateText = rendered.get(templateId);
+            if (templateText == null) {
+                throw new NoSuchElementException("No rendered template found for placeholder with id " + templateId);
```

Comment:
`templateId` passed in might not be a placeholder...

Reply:
Fixed